docs/scopes.md

```diff
@@ -0,0 +1,201 @@
+# OAuth Scopes
+
+Each authorization grant in OAuth has an associated "scope",
+a list containing one or more "scope values"
+that indicate what capabilities the granted token will have.
+Each individual scope value indicates a particular capability,
+such as the ability to read or write profile data,
+or to access the user's data in a particular service.
+
+As defined in [RFC6749 Section 3.3](https://tools.ietf.org/html/rfc6749#section-3.3),
+the scope of a token is expressed as
+a list of space-delimited, case-sensitive strings,
+and it is left up to the service
+to define the format and semantics
+of the individual scope values
+that make up this string.
+
+This document defines the scope values
+accepted in the Firefox Accounts ecosystem,
+and the rules for parsing and validating them.
+
+## Short-name scope values
+
+FxA supports a small set of "short-name" scope values
+that are identified by a short English word.
+These correspond either to
+scope values defined by external specifications (e.g. OpenID Connect),
+or to legacy scope values
+introduced during earily development.
+
+**No new short-name scope values should be added.**
+Instead we prefer to use URLs for new scope values,
```

Comment:
> +**No new short-name scope values should be added.**
> Instead we prefer to use URLs for new scope values,

Ah interesting 👍 . I'm surprised to hear this but i guess this makes sense after looking at Google's list of these scopes